lib/code_corps/messages/conversations.ex

```diff
@@ -0,0 +1,19 @@
+defmodule CodeCorps.Messages.Conversations do
+  @moduledoc ~S"""
+  Subcontext aimed at managing `CodeCorps.Conversation` records aimed at a
+  specific user belonging to a `CodeCorps.Message`.
+  """
+
+  alias Ecto.Changeset
+
+  alias CodeCorps.{Conversation}
+
+  @doc false
+  @spec create_changeset(Conversation.t, map) :: Ecto.Changeset.t
+  def create_changeset(%Conversation{} = conversation, %{} = attrs) do
+    conversation
+    |> Changeset.cast(attrs, [:user_id])
+    |> Changeset.validate_required([:user_id])
+    |> Changeset.assoc_constraint(:user)
+  end
```

Comment:
This changeset is needed to `cast_assoc`. The `message_id` is provided automatically by `cast_assoc` and is not part of the params.

Reply:
This will need a test.